apps/full-site-editing/full-site-editing-plugin/newspack-blocks/blog-posts-block-editor.js

```diff
@@ -1,25 +1,29 @@
 /* eslint-disable import/no-extraneous-dependencies */
 /**
- * Internal dependencies
+ * External dependencies
```

Comment:
we usually call these `WordPress dependencies`

Reply:
```suggestion
 * WordPress dependencies
```